### PR TITLE
get_user_group function to find get all groups of a user

### PR DESCRIPTION
**Pull request by individual-it**
simple function to retrieve all groups of a user

### CODE REVIEW

File: owncloud/owncloud.py [hunk 1]
```diff
@@ -855,6 +855,30 @@ def add_user_to_group(self, user_name, group_name):
...
+
+        :param user_name:  user to be checked
+        :returns: list of groups
+        :raises: ResponseError in case an HTTP error status was returned
+
+        """
+        res = self.__make_ocs_request(
+            'GET',
+            self.OCS_SERVICE_CLOUD,
+            'users/' + user_name + '/groups'
```
Comment (PVince81) [inline owncloud/owncloud.py]: can you escape the user_name? (just in case)

**Comment by individual-it** [comment]
you are not doing it anywhere else. Why here?

**Comment by PVince81** [comment]
Doing it here for file paths: https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L1291

If you find other locations then maybe encoding/escaping was forgotten.

Ok, you convinced me to ignore this. As a result, I now merged an earlier PR from someone with the same problem: https://github.com/owncloud/pyocclient/pull/105

Please rebase yours then I'll merge it.

**Comment by PVince81** [comment]
Raised https://github.com/owncloud/pyocclient/issues/109 to check all occurrences where escaping might have been missing already.

**Comment by PVince81** [comment]
Wait a minute, it seems this was already implemented in that other PR https://github.com/owncloud/pyocclient/pull/105/files#diff-22caf9e094e437c53f2b682ca2e48ef8R882

**Comment by PVince81** [comment]
I'll close your PR then.

If for any reason you're not happy with the other implementation, feel free to resubmit a PR with your improvements :smile: 

Thanks.

**Pull request closed without merging**